tools/ovc/openvino/tools/ovc/convert_impl.py

```diff
@@ -491,8 +492,22 @@ def _convert(cli_parser: argparse.ArgumentParser, args, python_api_used):
 
         argv.framework = model_framework
 
+        orig_input_model = argv.input_model
+        pytorch_model_on_disk = False
+        if argv.framework is None and get_pytorch_decoder_for_model_on_disk(argv, args):
```

Comment:
What if `argv.framework` will be "pytorch"?

Reply:
argv.framework is not external parameter of `ovc` and `convert_model`. If it is not `None` it means that framework is deduced in the code previous so we don't need to perform deducing here.